hw3/css/stylehw3.css

```diff
@@ -0,0 +1,477 @@
+* {
+	box-sizing: border-box;
+}
+body{
+	font: 14px/26px Arial, sans-serif;
+}
+/*HEADER*/
+.header {
+	background-image: url(../img/header.png);
+	background-repeat: repeat-x;
+	box-shadow: 0px 1px 13px 0px rgba(0, 0, 0, 0.294);
+	position: relative;
+}
+.header__wrapper {
+	width: 80%;
+	margin: 0 auto;
+	padding: 14px 0 15px;
+	line-height: 39px;
+	color: #fff;
+	display: flex;
+	justify-content: space-between;
+	position: relative;
+}
+.header__logo {
+	font-size: 48px;
+	font-family: Dunkirk, sans-serif;
+	display: flex;
+	align-items: center;
+}
+.header__logo::before {
+	content: url(../img/logo.png);
+	display: block;
+} 
+.header__menu {
+	display: flex;
+}
+.header__menu__item{
+	margin-left: 25px;
+}
+.header__menu__link {
+	color: #ffffff;
+	font-size: 18px;
+	display: block;
+}
+.header__menu__link:hover {
+	color: #4147e8;
+}
+/*RESPONSIVE MENU*/
+.responsive-menu {
+	position: relative;
+	font-size: 40px;
+	padding-left: 15px;
+	display: none;
+	cursor: pointer;
+}
+.dropdown-menu {
+	position: absolute;
+	right: 0;
+	top: 0;
+	background-color: #2b2626;
+	z-index: 200;
+	display: none;
+	min-width: 150px;
+	border: 1px solid black;
+}
+.dropdown-menu__link {
+	display: block;
+	color: #fff;
+	font-size: 18px;
+	line-height: 25px;
+	padding: 8px;
+}
+.responsive-menu:hover .dropdown-menu {
+	display: block;
+}
+.dropdown-menu__item:hover {
+	background-color: #696060;
+}
+/*BANNER	*/
+.banner {
+	background-image: url(../img/background-banner.png);
+	background-size: cover;
+}
+.banner__wrapper {
+	width: 80%;
+	margin: 0 auto;
+	min-height: 373px;
+	position: relative;
+	display: flex;
+	justify-content: flex-end;
+} 
+.banner__page1 {
+	left:18.75%;
+	top: 9.4%;
+	position: absolute;
+}
+.banner__page2 {
+	left: 0;
+	top: 18%;
+	position: absolute;
+}
+.banner__page3 {
+	left:4.6%;
+	top:31.6%;
+	position:absolute;
+}
+.banner__page3::after {
+	content: url(../img/perp-shadow.png);
+	display: block;
+	width: 399px;
+}
+.banner__info {
+	width: 37.5%;
+	margin: 60px 0 20px;
+	display: flex;
+	text-align: justify;
+	flex-direction: column;
+	color: #fff;
+}
+.banner__info__title {
+	margin-bottom: 25px;
+	text-transform: capitalize;
+	font-size: 26px;
+	font-family: DroidSans, sans-serif;
+	text-align: left;
+	font-weight: bold;
+	text-shadow: 1px 1.732px 2px rgba(0, 0, 0, 0.235);
```

Comment:
1.732px and 0.235 did you get it from PSD? Please use integers or one decimal. In case some browsers don't support